src/magic.cpp

```diff
@@ -1051,6 +1051,34 @@ std::string spell::enumerate_targets() const
     return ret;
 }
 
+std::string spell::list_targeted_monster_ids() const
+{
+    if( type->targeted_monster_ids.empty() ) {
+        return "";
+    }
+    std::vector<std::string> all_valid_target_ids;
+    for( auto iter : type->targeted_monster_ids ) {
```

Comment:
```suggestion
    for( const mtype_id &iter : type->targeted_monster_ids ) {
```
this isn't technically an iter, so perhaps change the variable too